resources.js

```diff
@@ -1,15 +1,86 @@
 export class Media {
-  // TODO
+  constructor(title){
+    this._title = title;
+    this._isCheckedOut = false;
+    this._ratings = [];
+  }
+  
+  get title(){
+    return this._title;
+  }
+  
+  get isCheckedOut(){
+    return this._isCheckedOut;
+  }
+  
+  get ratings(){
+    return this._ratings;
+  }
+
+  getAverageRating(){
+    const curRatings = this._ratings;
+    const curRatingsLength = curRatings.length;
```

Comment:
Not a big issue but these 2 are not adding great value to the cleanliness but adding even more variables. I'd recommend to remove.

Reply:
Ok. I will remove them later.
I tried to write cleaner code. I understand you.